roles/docker-app/templates/docker-app.service.j2

```diff
@@ -22,7 +22,7 @@ ExecStartPre=/bin/bash -c "/usr/bin/docker container inspect {{ docker_app_name
                                {% if docker_app_extra_params %}
                                {{ docker_app_extra_params }} \
                                {% endif %}
-                               {{ docker_app_container_name }}"
+                               {{ docker_app_container_name }}{% if docker_app_container_params %} {{ docker_app_container_params }}{% endif %}"
```

Comment:
Is the if required, if default is the empty string - will the space mess things up if there by itself?

Reply:
The extra space at the end of the line wouldn't functionally mess up anything, but would still cause the template file to change when run against a host originally provisioned with the older version of the role thus leading to unnecessary restart of the service. I thought that by putting the extra "if" there the change would be more transparent to existing users of the role.